about.html

```diff
@@ -0,0 +1,200 @@
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.4/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <title>AI Brains</title>
+  </head>
+  <body>
+    <header class="hidden-desktop container-fluid p-0">
+      <div class="d-flex p-2">
+        <ul
+          class="container d-flex justify-content-end px-5 fs-6 m-0 gap-3 text-white"
+        >
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Log out</li>
+        </ul>
+      </div>
+      <div class="container-fluid bg-white d-flex justify-content-center">
+        <div class="container d-flex justify-content-between p-4 fs-6 m-0">
+          <a href="index.html"
+            ><img src="img/AIBrain.ico" alt="AI brain logo" width="30px" /> AI
+            <span>B</span>rains</a
+          >
+          <ul class="d-flex fs-6 align-items-center m-0 gap-3 p-0">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <!-- Main content -->
+    <main class="container-fluid p-0 text-center">
+      <!-- About intro section -->
+      <!-- Menu mobile -->
+      <div id="menu" class="hidden-mobile flex-column">
+        <div class="menu-mobile">
+          <button class="close-btn d-flex justify-content-start"><i class="bi bi-x-lg"></i></button>
+          <ul class="menu-options">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+      <img src="img/bg-intro-min.png" alt="AI background image" class="bg-image">
+      <section
+        class="aboutintro-container d-flex flex-column p-3 p-lg-5 align-items-center"
+      >
+        <button class="hidden-mobile menu-btn btn align-self-start p-2">
+          <img src="img/btn-menu.png" alt="menu mobile" width="24px" />
+        </button>
+        <span class="fs-3 mt-5 title">"Hello! Human world"</span>
+        <h1 class="mb-3 title fw-bolder">
+          AI brains <br />
+          Global Summit 2023
+        </h1>
```

Comment:
- [ ]  Great job working on the design so far but I think it would be great if you add more font size and font weight to the heading since in the given design it had more font size and font weight so kindly add it 💯🔥🎉

- Your design
![image](https://user-images.githubusercontent.com/58553711/235038162-a5ceb12a-f4f2-4662-a6a2-934941bc25d4.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/235038208-e389ba26-c0a5-4906-afc6-ed5378bfa145.png)